src/containers/EntryPage.js

```diff
@@ -97,7 +97,7 @@ class EntryPage extends React.Component {
   handlePersistEntry = () => {
     const { persistEntry, collection } = this.props;
     setTimeout(() => {
-      persistEntry(collection).then(() => this.handleCloseEntry());
+      persistEntry(collection).then(() => this.handleCloseEntry()).catch(() => {/* Don't do anything, the error will be shown by the action. */});
```

Comment:
I think the error here should continue to propagate, rather than be caught by the empty `catch` block, unless there's a particular reason you caught this here.

Reply:
The only reason I did this was that I figured since the action is already showing a notification, we didn't need to do anything. I'll go ahead and test it without the `.catch` and make sure that there are no side effects, then I can remove it.

Reply:
Looks like the only "downside" is that you get duplicate error messages in the console, so I'll go ahead and remove this.